application/models/user_test.go

```diff
@@ -821,6 +931,90 @@ func (ms *ModelSuite) TestUser_FindByID() {
 	}
 }
 
+func (ms *ModelSuite) TestUser_FindByEmail() {
+	t := ms.T()
+
+	f := createUserFixtures(ms.DB, 2)
+
+	tests := []struct {
+		name    string
+		email   string
+		wantErr string
+	}{
+		{
+			name:    "Good",
+			email:   f.Users[0].Email,
+			wantErr: "",
+		},
+		{
+			name:    "Bad",
+			email:   "bad@example.com",
+			wantErr: "sql: no rows in result set",
```

Comment:
I just learned about this built-in constant.
```suggestion
			wantErr: sql.ErrNoRows,
```

Reply:
:+1: ... it also needs the `.Error()`  on the end to make it a string.